crates/storage/provider/src/providers/chain_info.rs

```diff
@@ -132,7 +143,7 @@ struct ChainInfoInner {
     /// The canonical head of the chain.
     canonical_head: RwLock<SealedHeader>,
     /// The block that the beacon node considers safe.
-    safe_block: RwLock<Option<SealedHeader>>,
+    safe_block: watch::Receiver<Option<SealedHeader>>,
     /// The block that the beacon node considers finalized.
-    finalized_block: RwLock<Option<SealedHeader>>,
+    finalized_block: watch::Receiver<Option<SealedHeader>>,
```

Comment:
this is a bit redundant, because we can simply use the Sender here, because `Sender` can also `borrow` the current value and then we don't need any changes in ChainInfoTracker